### DISCUSSION TITLE
[NOT RELEVANT] React RFCs test selectors

### DISCUSSION THREAD

**Discussion by AugustinLF** [callstack/react-native-testing-library#991] [General]
FYI, there are some discussions on universal selectors used for tests. We might want to get involved since it does mention that it envisions itself as being platform/renderer agnostic

https://github.com/reactjs/rfcs/pull/221

**Comment by thymikee** [comment]
I've had a look at this, and it makes sense for e2e libraries. We'd need to change the whole idea of the library, running the code on real devices. So I don't see how this would fit RNTL. Different scope

**Reply by AugustinLF** [reply]
Yes you're right. I initially thought it would still be relevant for us but does seem to be only E2E.
